packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -258,6 +258,14 @@ class CreateCommand extends CreateBase {
         'template: the language will always be C or C++.',
         exitCode: 2,
       );
+    } else if (!generateModule && argResults!.wasParsed('ios-language')) {
+      // Swift is not yet supported for add-to-app https://github.com/flutter/flutter/issues/23955.
+      globals.printWarning(
+          'The "ios-language" option is deprecated and will be removed in a future Flutter release.');
+      if (stringArg('ios-language') == 'objc') {
```

Comment:
Do we want this call to action for all cases, or just plugins? I was envisioning just the latter, since the usage is higher there.

Reply:
I figured I would do it for all in case there's some super compelling reason for the app template as well that I haven't thought of.  This should be a rare warning.

I did remove the module check since no one should be using the flag for that template anyway, may as well warn them it's being removed if they are using it for some weird reason.